### PR TITLE
Basel-Stadt, Switzerland

### PR DESCRIPTION
**Pull request by sergiyprotsiv**
Here are the addresses in the Swiss canton of Basel-Stadt. As with many sources in Switzerland, the canton has a "geoshop" where one can order the data. It then comes to the mailbox as a link to download the file (which I then upload to S3).

### CODE REVIEW

**Comment by sergiyprotsiv** [comment]
Re-run this please.

**Comment by sergiyprotsiv** [comment]
Re-run this please.

**Comment by sergiyprotsiv** [comment]
Something is wrong either with file encoding or with how machine is treating it: even after providing "latin1" as the encoding in the conform it tries to parse it as utf-8.

**Comment by migurski** [comment]
I think this is related to https://github.com/openaddresses/machine/issues/512. I have a fix completed, but it might be a bit fiddly to deploy so I want to make sure I do it when I have 20-30 minutes set aside to get the sequencing right.

**Comment by migurski** [comment]
Re-run this please.

**Comment by migurski** [comment]
Okay that worked, though it does look a bit offset. Setting an SRS might help.

File: sources/ch/basel-stadt.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
         "number": "NAUSNR",
```
Comment (thatdatabaseguy) [inline sources/ch/basel-stadt.json]: Think the field name is HAUSNR

**Pull request merged**